main/src/freertos_main.cpp

```diff
@@ -161,13 +175,13 @@ extern "C" void freertos_main()
     /* Initialize LVGL (Light and Versatile Graphics Library) and other resources */
 
     /* Create the LVGL task */
-    if (xTaskCreate(lvgl_task, "LVGL Task", 4096, nullptr, 1, nullptr) != pdPASS) {
+    if (xTaskCreate(lvgl_task, "LVGL Task", 4096, nullptr, 3, nullptr) != pdPASS) {
```

Comment:
```suggestion
    if (xTaskCreate(lvgl_task, "LVGL Task", (1 * 1024 * 1024), nullptr, 3, nullptr) != pdPASS) {
```

Any reason to change the task priority ?